src/util/Path.cpp

```diff
@@ -1,20 +1,14 @@
 #include "Path.h"
+
 #include "StringUtils.h"
 
 #include <glib/gstdio.h>
-#include <string.h>
 
-Path::Path()
-{
-}
-
-Path::Path(const Path& other)
- : path(other.path)
-{
-}
+#include <utility>
+#include <cstring>
```

Comment:
properly group imports if you are refactoring them anyway:
 - at the beginning: corresponding header
 - first group STL
 - second group other libs (eg glib)
 - third group project files

Reply:
Right that's the common standard, but in this project it is:
- at the beginning: corresponding header
- first group project files
- second group other libs (eg glib)
- third group STL